python-project/test/test_image.py

```diff
@@ -50,6 +50,14 @@ def test_threshold() -> None:
         output.save(Path(temp, 'image.png'))
 
 
+def test_threshold_rgb_rejection() -> None:
+    imageRGB = Image.new('RGB', (300, 200), (228, 150, 150))
+    threshold = Threshold()
+    outputRGB = threshold.run(imageRGB)
+    print(outputRGB)
```

Comment:
Is there a reason you want to print this in the test output?

Reply:
Removed, mistake